synapse/cmds/cortex.py

```diff
@@ -87,15 +87,37 @@ def nodevalu(t):
             return resp
 
         for node in nodes:
+
             form = node[1].get('tufo:form')
             valu = node[1].get(form)
 
-            tags = sorted(s_tufo.tags(node,leaf=True))
-            tags = [ '#'+tag for tag in tags ]
+            leafs = set(sorted(s_tufo.tags(node,leaf=True)))
+
+            taglines = []
+            for tag in sorted(s_tufo.tags(node)):
```

Comment:
There is no coverage for this block - a test which makes a tufo and throws a timeboxed tag and a non-timeboxed tag 
 & validates the output should provide coverage.